ntc_templates/templates/fsas_sir_show_system_information.textfsm

```diff
@@ -0,0 +1,29 @@
+Value VERSION (\d\d.\d\d)
+Value SERIAL (\d+)
+Value SYSTEM (.+)
+Value FIRMWARE (firmware\d)
+Value CONFIG_FILE (config\d)
+Value STARTUP_TIME (.+)
+Value CURRENT_TIME (.+)
+Value FIRMWARE1_VERSION (\d\d.\d\d)
+Value FIRMWARE2_VERSION (\d\d.\d\d)
+
+Start
+  ^Current-time\s+:\s+${CURRENT_TIME}\s*$$
+  ^Startup-time\s+:\s+${STARTUP_TIME}\s*$$
+  ^System\s+:\s+${SYSTEM}\s*$$
+  ^Serial No.\s+:\s+${SERIAL}\s*$$
+  ^ROM Ver.\s+:\s+.+$$
+  ^Firm Ver.\s+:\s+V${VERSION}\s*.+$$
+  ^Running-firmware\s+:\s+${FIRMWARE}\s*$$
+  ^Firmware1 Ver.\s+:\s+V${FIRMWARE1_VERSION}\s*.+$$
+  ^Firmware2 Ver.\s+:\s+V${FIRMWARE2_VERSION}\s*.+$$
+  ^Security Software Ver.\s+:\s+.+$$
```

Comment:
Match whitespace with regexes just in case the spacing ever changes. Flexibility should keep the care-and-feeding of the template to a minimum in the long run.

Escape certain period characters to match literal periods.

```suggestion
  ^Serial\s+No\.\s+:\s+${SERIAL}\s*$$
  ^ROM\s+Ver\.\s+:\s+.+$$
  ^Firm\s+Ver\.\s+:\s+V${VERSION}\s*.+$$
  ^Running-firmware\s+:\s+${FIRMWARE}\s*$$
  ^Firmware1\s+Ver\.\s+:\s+V${FIRMWARE1_VERSION}\s*.+$$
  ^Firmware2\s+Ver\.\s+:\s+V${FIRMWARE2_VERSION}\s*.+$$
  ^Security\s+Software\s+Ver\.\s+:\s+.+$$
```